src/System.Xml.XPath.XDocument/src/packages.config

```diff
@@ -1,13 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="System.Runtime" version="4.0.20-beta-22512" />
-  <package id="System.Xml.ReaderWriter" version="4.0.10-beta-22512" />
-  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22512" />
-  <package id="System.Globalization" version="4.0.10-beta-22512" />
-  <package id="System.Diagnostics.Tools" version="4.0.0-beta-22512" />
   <package id="System.Diagnostics.Debug" version="4.0.10-beta-22512" />
+  <package id="System.Diagnostics.Tools" version="4.0.0-beta-22512" />
+  <package id="System.Globalization" version="4.0.10-beta-22512" />
+  <package id="System.IO" version="4.0.10-beta-22512" targetFramework="portable-net45+win" />
+  <package id="System.Linq" version="4.0.0-beta-22512" />
   <package id="System.Reflection" version="4.0.10-beta-22512" />
-  <package id="System.Threading" version="4.0.0-beta-22512" />
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22512" />
+  <package id="System.Runtime" version="4.0.20-beta-22512" />
   <package id="System.Runtime.Extensions" version="4.0.10-beta-22512" />
-  <package id="System.Linq" version="4.0.0-beta-22512" />
+  <package id="System.Text.Encoding" version="4.0.10-beta-22512" targetFramework="portable-net45+win" />
+  <package id="System.Threading" version="4.0.0-beta-22512" />
+  <package id="System.Threading.Tasks" version="4.0.10-beta-22512" targetFramework="portable-net45+win" />
```

Comment:
These dependencies were added by nuget.

1. `System.Xml.XPath.XDocument` depends on **`System.Xml.XDocument`** and `System.Xml.XPath`
2. **`System.Xml.XDocument`** depends on `System.IO` and **`System.Xml.ReaderWriter`**
3. **`System.Xml.ReaderWriter`** depends on `System.Text.Encoding` and `System.Threading.Tasks`